support.md

```diff
@@ -1,19 +0,0 @@
----
-title: Support Resources
-summary: Find support resources for your TiDB installation.
```

Comment:
@lilin90 PTAL whether we should keep this file. It is shown in https://docs.pingcap.com/tidb/dev/support but not in the TOC menu. Thanks~

Reply:
@TomShawn Most of these resources are also displayed in the footer, so I agree with deleting this file. Let's focus on documentation content in `.md` files.

Reply:
Get. Thanks!